docs/general/administration/troubleshooting.md

```diff
@@ -130,3 +130,49 @@ After opening the database, navigate to the Execute SQL Tab and execute the foll
 UPDATE Users SET InvalidLoginAttemptCount = 0 WHERE Username = 'LockedUserName';
 UPDATE Permissions SET Value = 0 WHERE Kind = 2 AND UserId IN (SELECT Id FROM Users WHERE Username = 'LockedUserName');
 ```
+
+## Fix Admin User Permissions
+
+If the Permissions for your Admin Account break, you can Restore them using simple SQL Queries.
+
+:::caution
+Manual changes to the database can destroy your Instance beyond repair. to prevent this create a copy of your database before executing:
+`cp /PATH/TO/JELLYFIN/DB/jellyfin.db  /PATH/TO/JELLYFIN/DB/jellyfin.db.bck`
+:::
+
+Before continuing, make sure that you have sqlite3 installed.
+When sqlite3 is not installed, you can install it under Debian based systems with `apt install sqlite3`.
+After that do the following commands/SQL query:
+
+```bash
+sqlite3 /PATH/TO/JELLYFIN/DB/jellyfin.db
+```
+
+### Get an Overview
+
+To see all your current permissions for all users, you can run the following Query:
+
+```sql
+select Permissions.Value,Permissions.Kind,Users.Username  from Permissions Inner Join Users on Permissions.UserID = Users.Id;
```

Comment:
You have different caseings in all 3 of the SQL queries. I recommend all uppercase

Reply:
Changed it, thanks